src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -89,6 +91,29 @@
   
   </ItemGroup>
 
+  <!--**********************************************************************-->
+  <!-- Workarounds -->
+  <!--**********************************************************************-->
+
+  <!-- The OWIN assemblies aren't being included in the VSIX for some reason.	
+       This workaround adds metadata to three artifacts so that the standard VSSDK targets will	
+       include them.	
+       NB the target is based on the one from Microsoft.VisualStudio.SDK.EmbedInteropTypes.
+       We have a similar hack in Intergration.csproj -->
```

Comment:
I've added #1832 to the infrastructure backlog to investigate whether we can clean up the VSIX authoring.